File: 8/victor_gaitour/picmaker.py
s="P3\n512 512\n 255\n"
s1=""
for k in range(4):
    for i in range(256):
        for j in range(256):
            s1=s1+str(i)+" "+str(j)+" "+str(k*64)+" "

with open("pic.ppm","w") as f:
    f.write(s+s1)
    f.close

File: 4/jenny_baran/picmaker.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <fcntl.h>
#include <unistd.h>

int main() {
    int fd;
    int r, g, b;
    char line[256];
    
    fd = open( "pic.ppm", O_CREAT | O_TRUNC | O_WRONLY, 0664 );
    
    sprintf( line, "P3\n500 500\n255\n" );
    write( fd, line, strlen(line) );
    
    for( int i = 0; i < 500; i++ ) {
        for( int j = 0; j < 500; j++ ) {
            r = 255;
            g = j%255;
            b = i%255;
            
            sprintf( line, "%d %d %d ", r, g, b );
            write( fd, line, strlen(line) );
        }
    }
    
    close( fd );
    
    return 0;
}